playbooks/common/openshift-cluster/evaluate_groups.yml

```diff
@@ -29,6 +29,10 @@
       msg: The nfs group must be limited to one host
     when: "{{ (groups[g_nfs_hosts] | default([])) | length > 1 }}"
 
+  - fail:
```

Comment:
I wasn't sure when I started writing this, I was mostly following the patterns the NFS functionality had done. I talked to @abutcher about this, but I've already forgotten the distinction between `l_`, `g_`, and `oo_` variables, my apologies. :) Is there any meaningful distinction?

Realistically this task could probably be removed since the definition of the var being tested is `g_glusterfs_hosts: "{{ groups.glusterfs | default([]) }}"`, unless this playbook is intended to be able to run without first running `cluster_hosts.yml`